### PR TITLE
Start using next seo in claim page#307

### PR DESCRIPTION
**Pull request by pepermao**
https://seoanalyzer.me/resources/pdf-reports/ea4b-138-97-120-243.sa.ngrok.io_438059.pdf

### CODE REVIEW

File: src/pages/claim-page.tsx [hunk 1]
```diff
@@ -1,11 +1,53 @@
...
+            }
+        })
+    })
+
+    
     return (
-        <ClaimView personality={personality} claim={claim} href={href}/>
+        <>
+            <Head>
+                <script type="application/ld+json">{`
```
Comment (thesocialdev) [inline src/pages/claim-page.tsx:27]: Nice! But let's do this on a separate PR and use another issue for it. I have a lot of considerations for this feature.

File: server/root/root.controller.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { Controller, Get, Req, Res } from "@nestjs/common";
 
-@Controller("api")
+@Controller()
```
Comment (thesocialdev) [inline server/root/root.controller.ts:3]: The health endpoint should still have the `api` prefix, otherwise it will break how `kubernetes` check for the service health in production.

File: src/components/Personality/PersonalityCard.tsx [hunk 3]
```diff
@@ -49,7 +49,7 @@ const PersonalityCard = ({
 
     if (personality) {
         return (
-            <Row
+            <section
```
Comment (thesocialdev) [inline src/components/Personality/PersonalityCard.tsx:52]: Same as above

File: public/locales/pt/seo.json [hunk 4]
```diff
@@ -2,5 +2,10 @@
     "personalityDescription": "Veja o perfil de {{name}} na AletheiaFact.org",
     "personalityImageAlt": "Imagem de {{name}}",
     "personalityListTitle": "Perfis de personalidades",
-    "personalityListDescription": "Veja todos os perfis na AletheiaFact.org"
+    "personalityListDescription": "Veja todos os perfis na AletheiaFact.org",
```
Comment (thesocialdev) [inline public/locales/pt/seo.json:5]: Missing English versions?

**Review by thesocialdev — CHANGES_REQUESTED** [review]

File: server/root/root.controller.ts (hunk 2 above)
Comment (pepermao) [reply server/root/root.controller.ts:3]: but with api prefix we run into SEO problem

image

Comment (thesocialdev) [reply server/root/root.controller.ts]: Let's put the prefix only for the health endpoint
